modules/ROOT/pages/how_to_contribute.adoc

```diff
@@ -0,0 +1,41 @@
+= How to Contribute to the ownCloud Documentation
+
+We welcome all helpful contributions to the ownCloud documentation, whether they’re reporting issues, adding new material, or writing tips and tricks. 
+
+If you wish to contribute to the official _Administration_, _User_, _Desktop Client_, and _Mobile Apps_ manuals, you must:
+
+* Have a xref:https://github.com/[GitHub] account.
+* Fork the official repositories, write your material, and create a pull request. 
+
+The manuals are written using xref:https://docs.antora.org/antora/[Antora] and xref:https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/[the AsciiDoc file format].
+
+* The Administration, Developer, and User manuals are in xref:https://github.com/owncloud/docs/[the owncloud/docs repository on GitHub].
+  Read xref:https://github.com/owncloud/docs/blob/master/docs/getting-started.md[the documentation] for instructions on setting up your build environment and becoming familiar with the file format and build system that power the documentation.
+* The documentation for the desktop sync client is in the doc directory of the Desktop Client repo. 
+  See xref:https://github.com/owncloud/client/blob/master/README.md[its README] for instructions.
+* The documentation of the Android app is in the user_manual directory of the Android App repo. 
+  See xref:https://github.com/owncloud/android/blob/master/README.md[its README] for instructions.
+* The documentation of the iOS app is in the user_manual directory of the iOS App repo. 
+  See xref:https://github.com/owncloud/ios/blob/master/README.md[its README] for instructions.
+
+== Reporting Documentation Errors
+
+If you find errors or omissions in the _Administration_, _User_, _Desktop Client_, and _Mobile Apps_ manuals, we encourage you to report them. 
+
+NOTE: The Developer manual has custom requirements; please consult the Developer manual for details.
+
+Please follow this template when reporting documentation errors:
+
+. First, search the appropriate GitHub repository for matching or similar issues.
+. If a matching or similar issue is not available, open a new Issue in the proper repository.
+. Provide the URL to the manual page and section. 
+  There are multiple release versions of most of the manuals, so please be specific as to which version needs correcting, and if the issue is in the HTML or PDF version.
+. Quote the particular error.
+. Provide the correct solution (if you know it) and links to supporting references. 
+  If you don't know the answer that is OK.
+. Provide screenshots if they add useful information.
+. Provide your web browser, its version, and your operating system & version.
+
+== Reporting Documentation Suggestions
+
+If you don't have a specific problem or contribution, but wish to offer a suggestion, open an Issue in the appropriate repository.
```

Comment:
If you have a doc suggestion you do not file it in eg core - or?

Reply:
On reading the changes in the PR through again, I see that the content could be trimmed down somewhat. However, regarding "the appropriate repository", in this context, it's referring to the Administration, Developer, User, Desktop Client, and Mobile App manuals, not core.